src/main/java/frc/robot/config/FluidConstant.java

```diff
@@ -75,14 +75,8 @@ public T get() {
     public T getValue() {
         return this.value;
     }
+
     
-    /**
-     * Get method for implementing the Supplier interface

```

Comment:
The duplicate get method has been resolved in Justin's commit and you have no other changes in FluidConstants. Since this is showing as a merge conflict on your pull request I would just remove this file from your pull request.